### PR TITLE
Improvement: Convert quotes intention and annotation for interpolated strings with wrong quotes

### PR DESCRIPTION
**Pull request by mayakwd**
- Added check for expressions with interpolated strings wrapped by wrong quotes
- Added convert quotes intention

### CODE REVIEW

File: src/common/com/intellij/plugins/haxe/ide/intention/ConvertQuotesIntention.java [hunk 1]
```diff
@@ -0,0 +1,97 @@
...
+  private HaxeStringLiteralExpression expression;
+
+  public ConvertQuotesIntention() {
+  }
+
+  @Nls
+  @NotNull
+  @Override
+  public String getFamilyName() {
+    return "Convert quotes";
```
Comment (EricBishton) [inline src/common/com/intellij/plugins/haxe/ide/intention/ConvertQuotesIntention.java]: I noticed that you started using property bundles in your latest submissions.  Do you want to go back and do that in this file?  (In another submission.  I'm going to pull this one.)

**Comment by EricBishton** [comment]
You might also want to start giving some thought to unit testing.  There are quite a number of unit tests around semantic annotation and intentions already in the testSrc directory that you could use as examples.

File: src/common/com/intellij/plugins/haxe/ide/intention/ConvertQuotesIntention.java (hunk 1 above)
Comment (mayakwd) [inline src/common/com/intellij/plugins/haxe/ide/intention/ConvertQuotesIntention.java]: @EricBishton I'm going to move every string in bundle - and I'll make new pull request.

**Comment by mayakwd** [comment]
Yep, forget about tests, and I'll fix it too :-)

**Pull request merged**